UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -68,67 +73,95 @@ private void OnValidate()
 		counting = true;
-		restartTime = 10f;
+        ShuttleCounting = false;
 		UpdateRoundTimeMessage.Send(GetRoundTime);
 	}
 
-	private void Update()
-	{
-		if (waitForRestart)
-		{
+    public void ReportScores()
+    {
+        foreach (ConnectedPlayer player in PlayerList.Instance.InGamePlayers)
+        {
+            if (GetComponent<HealthBehaviour>().IsDead == false)
+            {
+                LifeCount++;
+                PostToChatMessage.Send(player.Name + " has survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+            }
+            if (LifeCount == 0)
+            {
+                PostToChatMessage.Send("No one has survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have killed everyone and won.", ChatChannel.OOC);
+            }
+            if (LifeCount == 1)
+            {
+                PostToChatMessage.Send(LifeCount + " person survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have failed to detonated the bomb.", ChatChannel.OOC);
+            }
+            if (LifeCount > 1)
+            {
+                PostToChatMessage.Send(LifeCount + " people survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have failed to detonated the bomb.", ChatChannel.OOC);
+            }
+        }
 
-			restartTime -= Time.deltaTime;
-			if (restartTime <= 0f)
-			{
-				waitForRestart = false;
-				RestartRound();
-			}
-		}
+    }
 
-		else if (counting)
+    private void Update()
+	{
+		if (counting)
 		{
 			GetRoundTime -= Time.deltaTime;
 			roundTimer.text = Mathf.Floor(GetRoundTime / 60).ToString("00") + ":" +
-			                  (GetRoundTime % 60).ToString("00");
+			                  (GetRoundTime % 60).ToString("00") + " ETA";
 			if (GetRoundTime <= 0f)
 			{
 				counting = false;
-				roundTimer.text = "GameOver";
-				
-				// Prevents annoying sound duplicate when testing
-				if (SystemInfo.graphicsDeviceType != GraphicsDeviceType.Null && !GameData.Instance.testServer)
-				{
-					SoundManager.Play("ApcDestroyed", 0.3f, 1f, 0f);
-				}
-
-				if (CustomNetworkManager.Instance._isServer)
-				{
-					waitForRestart = true;
-					PlayerList.Instance.ReportScores();
-				}
+                ShuttleCounting = true;
 			}
 		}
-	}
+        else if (ShuttleCounting)
+        {
+            GetShuttleTime -= Time.deltaTime;
+            roundTimer.text = Mathf.Floor(GetShuttleTime / 60).ToString("00") + ":" +
+                              (GetShuttleTime % 60).ToString("00") + " ETD";
+
+            if (GetShuttleTime <= 0f)
+            {
+                ShuttleCounting = false;
+                RestartRound();
+            }
+            else if (GetShuttleTime <= 10f)
+            {
+                ReportScores();
```

Comment:
This is called hundreds of times because there is no condition on only reporting the score once.

Reply:
For some reason it didn't push my latist shit. Pushed again. fixed.